playbooks/common/openshift-master/config.yml

```diff
@@ -154,6 +154,7 @@
   - role: openshift_clock
   - role: openshift_cloud_provider
   - role: openshift_builddefaults
+  - role: openshift_buildoverrides
```

Comment:
should this role actually be a dependency of the openshift_master role (or openshift_master_facts) rather than a separate role? It seems like it is a dependency, but is only being enforced by the playbook and not the role metadata.